catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -195,6 +192,10 @@ public void create() {
 		virtualWidthHalf = virtualWidth / 2;
 		virtualHeightHalf = virtualHeight / 2;
 
+		font = new BitmapFont();
+		font.setColor(1f, 0f, 0.05f, 1f);
+		font.getData().setScale(1.2f);
```

Comment:
since hard coded values should be avoided, and i guess its important that its the same value as in line 633, please put this in a member variable and give it a proper name.